tensorboard/plugins/graph/graphs_plugin_test.py

```diff
@@ -198,41 +239,75 @@ def test_graph_large_attrs(self):
     self.assertEqual({'message_prefix': [b'value']},
                      large_attrs)
 
-  def test_run_metadata(self):
-    self.set_up_with_runs()
-    (metadata_pbtxt, mime_type) = self.plugin.run_metadata_impl(
+  @with_runs()
+  def test_run_metadata(self, plugin):
+    (metadata_pbtxt, mime_type) = plugin.run_metadata_impl(
         self._RUN_WITH_GRAPH, self._METADATA_TAG)
     self.assertEqual(mime_type, 'text/x-protobuf')
     text_format.Parse(metadata_pbtxt, config_pb2.RunMetadata())
     # If it parses, we're happy.
 
-  def test_is_active_with_graph_without_run_metadata(self):
-    self.generate_run('w_graph_wo_meta',
+  @with_runs()
+  def test_is_active_with_graph_without_run_metadata(self, plugin):
+    logdir = self.get_temp_dir()
+    self.generate_run(logdir,
+                      'w_graph_wo_meta',
                       include_graph=True,
                       include_run_metadata=False)
-    self.bootstrap_plugin()
-    self.assertTrue(self.plugin.is_active())
-
-  def test_is_active_without_graph_with_run_metadata(self):
-    self.generate_run('wo_graph_w_meta',
+    self.bootstrap_plugin(logdir)
```

Comment:
I’m not following; isn’t this a no-op? This creates, populates, and
returns a new multiplexer, but the client (here) discards that because
it already has a `plugin`. Can we drop this line in each test case?

Reply:
Cleaned up; see above.